piv/piv.go

```diff
@@ -176,6 +180,12 @@ func (c *client) Open(card string) (*YubiKey, error) {
 		return nil, fmt.Errorf("getting yubikey version: %w", err)
 	}
 	yk.version = v
+
+	// If yubikey is 5.7.0 or newer, management key default is AES192
+	if int(v.major) >= 5 && int(v.minor) >= 7 {
+		DefaultManagementKeyType = algAES192
```

Comment:
can this be on the client rather than global to the package?

Reply:
I tested with v4.3.7, v5.4.3, and v5.7.1 yubikeys. It has been possible for users to change the type of management key since 5.4, but the default has remained 3DES. I decided in lieu of this minimal fix, I would take things a step further and just implement dynamic support for all the supported management key types. Closing this PR in favor of : https://github.com/go-piv/piv-go/pull/149